### ISSUE TITLE
Suggestion: Owned actors only

### ISSUE THREAD

**Issue by Ator27** [1000nettles/combat-numbers#47]
Maybe consider only doing the "bounce" on actors that the player owns.  Not the other players tokens or NPC tokens.

**Comment by Fridan99** [comment]
Good idea. It would be great if there was an option so that only the GM and the players with owned actor could see the numbers. Obviously the GM should see EVERYTHING. As GM I like to see all numbers, but it is a pain that the players can see the hp modification of the npc.
